pkg/cloudprovider/fake/fake.go

```diff
@@ -30,6 +30,7 @@ type FakeCloud struct {
 	Err           error
 	Calls         []string
 	IP            net.IP
+	ID            string
```

Comment:
I can't name it with the same name of the method. I'll find something else (descriptive enough).